testplan/testing/multitest/base.py

```diff
@@ -603,10 +608,19 @@ def aborting(self):
         """Suppressing not implemented debug log from parent class."""
 
     def get_metadata(self) -> TestMetadata:
+
+        suites = []
+        for suite, testcases in self.get_test_context():
```

Comment:
`test_context` is property, inherited through `Test`, I would call it here because that would populate `_test_context` and no need to recalculate it again.